packages/backend/src/accounting/service.ts

```diff
@@ -407,7 +426,8 @@ async function createAccountWithdrawal(
       sourceAccountId: account.id,
       destinationAccountId: account.asset.unit,
       amount,
-      timeout
+      timeout,
+      ledger: account.asset.unit
```

Comment:
Should Rafiki's asset `unit` similarly be renamed `ledger`?

Reply:
@wilsonianb We should be good now on the `unit` -> `ledger` rename.